File: 1/A_1_11.c
// Болібрук Влад 
#include <stdio.h>
#include <stdlib.h>
#include <math.h>
int main() {
		
	int x = 1;
	int y = 2;
	int z = 3;

	float resultA = (sqrt(abs(x-1)) - pow(abs(y),1/3))/(1+(x*x)/2+(y*y)/4);
	float resultB = x*(atan(z)+ exp(-(x+3)));

	printf("task is A_1_11\n");
	printf("result is %f\n",resultA);
	printf("result is %f\n",resultB);

	return 0;	
}

File: S1/3/A_1_11.java
// Болібрук Влад 
public class A_1_11 {
	public static void main(String[] args) {
		int x = 1;
		int y = 2;
		int z = 3;

		float resultA = (float)((Math.sqrt(Math.abs(x - 1)) - Math.pow(Math.abs(y), 1 / 3)) / (1 + (x * x) / 2 + (y * y) / 4));
		float resultB = (float)(x * (Math.atan(z) + Math.exp(-(x + 3))));

		System.out.println("task is A_1_11");
		System.out.printf("result is %f\n", resultA);
		System.out.printf("result is %f\n", resultB);
	}
}